src/ui/Base/TopHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactNode } from 'react';
+
+const TopHeader = ({
+  label,
+  children,
+  ...props
+}: {
+  label: string;
+  children?: ReactNode;
+}) => {
+  const displayStyle = children ? 'flex justify-between items-center' : '';
+
+  return (
+    <div className={displayStyle}>
+      <p className="text-xl font-bold text-main-900" {...props}>
+        {label}
+      </p>
+      {children && children}
```

Comment:
children이 `undefined`인 경우 렌더링되지 않을테니 그냥 전달해도 좋을 것 같아요!

```suggestion
      {children}
```